amida-tech/indaba-client
# styling/inba-543-export-inactive

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
This fixes the inactive state that was not rendering because of a extended font. I also added pointers to the button and extended the lead font to the instructions.
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-543
#### How should this be manually tested?
Go to the export page and click on custom export or quick export. When one is selected, the other should have an opacity of 0.5 but hovering over inactive should bring opacity to 0.7
#### Background/Context

#### Screenshots (if appropriate):
